mlir-sycl/include/mlir/Dialect/SYCL/Transforms/Passes.td

```diff
@@ -27,10 +27,13 @@ def InlinePass : Pass<"inliner"> {
           "\"simple\", \"Inline callees using simple heuristics\"), "
         "clEnumValN(mlir::sycl::InlineMode::Aggressive, "
           "\"aggressive\", \"Inline callees using aggressive heuristics\"))">,
-    Option<"RemoveDeadCallees", "remove-dead-callees", "bool", /*default=*/"true",
+    Option<"RemoveDeadCallees", "remove-dead-callees", "bool", /*default=*/"false",
```

Comment:
I thought removing callees that are dead after inlining should be done by default. If they are dead then I do not see a good reason to leave them around unless one want to specifically overwrite the default. So IMO the original default makes more sense.

Reply:
Fine with me. I just that we were overriding this default and using `false` instead when creating the pass, so I thought it was a mistake here (50 % probability guess). Can I address this in another PR as this would imply changing some tests?

Reply:
Sure